bucket/bencode-editor.json

```
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://sites.google.com/site/ultimasites/bencode-editor",
+    "description": "A BEncode editor",
+    "version": "0.7.1.0",
+    "license": "Unknown",
+    "architecture": {
+        "64bit": {
+            "url": "https://sites.google.com/site/ultimasites/files/bencode-editor.0710x.zip",
+            "hash": "02c4e36834236021a0dbbbf4145355ea307843e8e5cc0c0e0b713da991ed6433"
+        },
+        "32bit": {
+            "url": "https://sites.google.com/site/ultimasites/files/bencode-editor.0710u.zip",
+            "hash": "63027f8b9a9320df2b9c20cc8885063bdd15a707ac77aa95673263a11dcdc454"
+        }
+    },
+    "shortcuts": [
```

https://sites.google.com/site/ultimasites/bencode-editor/history @ v0.3.0.0

> Feature: Load files from commandline

```suggestion
    "bin": [
        [
            "BEncode Editor.exe",
            "bencode-editor"
        ]
    ],
    "shortcuts": [
```